test/powershell/Modules/Microsoft.WSMan.Management/CredSSP.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+Describe "CredSSP cmdlet tests" -Tags 'Feature','RequireAdminOnWindows' {

```

Comment:
Why are these tests separated into separate files? We have been grouping tests for each cmdlet into a single file.